src/main/java/br/com/hackathon/repository/ItemRepository.java

```diff
@@ -0,0 +1,15 @@
+package br.com.hackathon.repository;
+
+import br.com.hackathon.domain.entity.ItemEntity;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+public interface ItemRepository extends CrudRepository<ItemEntity, Long> {
+
+    Optional<ItemEntity> findById(Long id);
+
+    @Override
+    List<ItemEntity> findAll();
```

Comment:
Acho que não precisa desse carinha aqui, ele já não vem por default ?

Reply:
Ele vem retornando Iterable. Mas como tá rolando casting na service dá pra tirar. Ou criamos um Wrapper de todos os repositories com esse override e tiramos o casting da service. To por vocês @EduardoThums @JPedroSilveira

Reply:
Creio que daria pra capar fora apenas

Reply:
Beleza